src/cascade/executor/no_covariate_main.py

```diff
@@ -207,9 +207,21 @@ def internal_model(model, inputs):
     for rate in config.non_zero_rates:
         getattr(model.rates, rate).parent_smooth = default_smoothing
 
+    avgint_rows = []
```

Comment:
We have both mains() to deal with, the no_covariate and the epiviz. I forgot that. Nice catch.